configure.ac

```diff
@@ -1051,6 +1051,45 @@
         fi
     fi
 
+  # WinDivert support
+    AC_ARG_ENABLE(windivert,
+        AS_HELP_STRING([--enable-windivert],[Enable WinDivert support [default=no]]),,
+        [enable_windivert="no"])
+    
+  # WinDivert can only be enabled on Windows builds
+    AC_CHECK_DECL([OS_WIN32],,[enable_windivert="no"])
```

Comment:
OS_WIN32 not set for CYGWIN. Probably not a problem as MinGW/native builds will be the new default from 4.1+.

Reply:
Would it be preferable to check for `$host in mingw32/cygwin" then?

Reply:
It's up to you. If you want to support CYGWIN as well it will have to change. But I don't mind limiting this new feature to MinGW/native win.